subgraphs/eigenlayer/src/mappings/handlers.ts

```diff
@@ -341,11 +377,34 @@ export function handleWithdrawalCompleted(event: WithdrawalCompleted): void {
     withdrawID,
     event
   );
+
+  const strategyContract = Strategy.bind(Address.fromBytes(poolID));
```

Comment:
Let's make this a helper function, that way if something changes we don't need to update 2 places and risk more bugs